docs/development.md

```diff
@@ -78,7 +85,12 @@ Then run `M-x add-dir-local-variable RET python-mode RET pyvenv-activate RET <pa
 ### Other setups
 
 - Install and enable [ShellCheck](https://github.com/koalaman/shellcheck) to find bash errors locally.
-- If you need to update the documentation, it is recommended to install the extension [Markdown All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one). It helps to maintain the table of contents in the documentation.
+- If need to update the documentation, it is recommended to install the extension
+  [Markdown All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one).
+  It helps to maintain the table of contents in the documentation.
+- While the black formatter does not handle long comments and the like
```

Comment:
How about provide a link to https://marketplace.visualstudio.com/items?itemName=stkb.rewrap, so it can be more actionable?